Beta_Integrations/ProofpointThreatResponse/ProofpointThreatResponse.py

```diff
@@ -1,22 +1,22 @@
 import demistomock as demisto
 from CommonServerPython import *
+
 ''' IMPORTS '''
-import requests
 import json
-requests.packages.urllib3.disable_warnings()
+import requests
+import dateutil.parser
+from datetime import date
 
-if not demisto.getParam('proxy'):
-    del os.environ['HTTP_PROXY']
-    del os.environ['HTTPS_PROXY']
-    del os.environ['http_proxy']
-    del os.environ['https_proxy']
+requests.packages.urllib3.disable_warnings()
 
 ''' GLOBAL VARS '''
-BASE_URL = demisto.params().get('url')
-if BASE_URL[-1] != '/':
-    BASE_URL += '/'
+BASE_URL = demisto.params().get('url', {})
+if BASE_URL:
+    if BASE_URL[-1] != '/':
+        BASE_URL += '/'
 API_KEY = demisto.params().get('apikey')
 VERIFY_CERTIFICATE = False
```

Comment:
Is the integration doesn't use `insecure` at all?

Reply:
Added the parameter